pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -40,10 +40,12 @@ import (
 // and perform translations from InTree PV's to CSI
 type InTreeToCSITranslator interface {
 	IsPVMigratable(pv *v1.PersistentVolume) bool
+	IsInlineMigratable(vol *v1.Volume) bool
```

Comment:
So the implementations were already implementing these functions?

Reply:
Yes. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/csi-translation-lib/translate.go#L197